papery/page.py

```diff
@@ -49,11 +50,20 @@ def text(self, lang=None, link=None):
         text = fp.read()
         fp.close()
 
+        html = markdown.markdown(text,
+                                 extensions=["tables", "pymdownx.emoji"],
+                                 extension_configs = {
+                                     "pymdownx.emoji": {
+                                         "emoji_index": pymdownx.emoji.gemoji,
+                                         "emoji_generator": pymdownx.emoji.to_png,
+                                         "alt": "short",
+                                         "options": {
+                                             # TODO make "image_path" configurable by papery's configuration file for security reasons
```

Comment:
I don't want to load emoji files from a CDN for security reasons especially for internal sites.